src/site/_data/design/tokens.json

```diff
@@ -0,0 +1,252 @@
+{
+  "colors": [
+    {
+      "group": "Core",
+      "items": [
+        {
+          "name": "Primary",
+          "hex": "#3740ff",
+          "hsl": "hsl(237, 100%, 61%)"
+        },
+        {
+          "name": "Primary Glare",
+          "hex": "#6a71ff",
+          "hsl": "hsl(237, 100%, 71%)"
+        },
+        {
+          "name": "Secondary",
+          "hex": "#e51661",
+          "hsl": "hsl(338, 82%, 49%)"
+        },
+        {
+          "name": "Secondary Glare",
+          "hex": "#ee518a",
+          "hsl": "hsl(338, 82%, 63%)"
+        },
+        {
+          "name": "Tertiary",
+          "hex": "#6001ff",
+          "hsl": "hsl(262, 100%, 50%)"
+        },
+        {
+          "name": "Tertiary Glare",
+          "hex": "#a067ff",
+          "hsl": "hsl(262, 100%, 70%)"
+        },
+        {
+          "name": "Brand",
+          "hex": "#3fc4ff",
+          "hsl": "hsl(198, 100%, 62%)"
+        }
+      ]
+    },
+    {
+      "group": "State",
+      "items": [
+        {
+          "name": "Good",
+          "hex": "#018642",
+          "hsl": "hsl(149, 99%, 26%)"
+        },
+        {
+          "name": "Good Bright",
+          "hex": "#0CCE6B",
+          "hsl": "hsl(149, 89%, 43%)"
+        },
+        {
+          "name": "Good Glare",
+          "hex": "#e2faed",
+          "hsl": "hsl(148, 71%, 93%)"
+        },
+        {
+          "name": "Warn",
+          "hex": "#D04900",
+          "hsl": "hsl(21, 100%, 41%)"
+        },
+        {
+          "name": "Warn Bright",
+          "hex": "#FFA400",
+          "hsl": "hsl(39, 100%, 50%)"
+        },
+        {
+          "name": "Warn Glare",
+          "hex": "#fff5e0",
+          "hsl": "hsl(41, 100%, 94%)"
+        },
+        {
+          "name": "Bad",
+          "hex": "#EB0F00",
+          "hsl": "hsl(4, 100%, 46%)"
+        },
+        {
+          "name": "Bad Bright",
+          "hex": "#FF4E42",
+          "hsl": "hsl(4, 100%, 63%)"
+        },
+        {
+          "name": "Bad Glare",
+          "hex": "#ffe9e8",
+          "hsl": "hsl(3, 100%, 95%)"
+        }
+      ]
+    },
+    {
+      "group": "Shades",
+      "items": [
+        {
+          "name": "Dark",
+          "hex": "#191919",
+          "hsl": "hsl(0, 0%, 10%)"
+        },
+        {
+          "name": "Gray",
+          "hex": "#545454",
+          "hsl": "hsl(0, 0%, 33%)"
+        },
+        {
+          "name": "Gray Bright",
+          "hex": "#8d8d8d",
+          "hsl": "hsl(0, 0%, 55%)"
+        },
+        {
+          "name": "Gray Glare",
+          "hex": "#d9d9d9",
+          "hsl": "hsl(0, 0%, 85%)"
+        },
+        {
+          "name": "Light",
+          "hex": "#f3f3f3",
+          "hsl": "hsl(0, 0%, 95%)"
+        },
+        {
+          "name": "Light Bright",
+          "hex": "#ffffff",
+          "hsl": "hsl(0, 0%, 100%)"
+        }
+      ]
+    }
+  ],
+  "textSizes": [
+    {
+      "name": "Base",
```

Comment:
Just curious, how did you arrive at the type sizes? Were these based on Jiwoong's mocks?
I was also wondering if you plan to have type classes that match the material naming like "body 1", "body 2", "caption", "eyebrow", etc.?

Reply:
Yeh these are from the mocks. We could create utils for those for sure, but I try to keep text sizes as neutral contextually as possible